rest/data/project.go

```diff
@@ -312,6 +312,22 @@ func (pc *DBProjectConnector) GetProjectSettingsEvent(p *model.ProjectRef) (*mod
 	return &projectSettingsEvent, nil
 }
 
+func (pc *DBProjectConnector) GetProjectAliasResults(p *model.Project, alias string, includeDeps bool) ([]restModel.APIVariantTasks, error) {
+	projectAliases, err := model.FindAliasInProject(p.Identifier, alias)
```

Comment:
Do we want to explicitly error if no projectAliases match, so the user understands that the input is the problem and not that there are no tasks/variants?

Reply:
yep, done